integration_tests/snapshots/log-go

```diff
@@ -0,0 +1,10 @@
+[log] {"message":{"message":"START XXX"}}
```

Comment:
What function is this snapshot (`log-go`) for? Is this from an older function that you deleted?